powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/service/behavior/tasks/CallbackUrlBehavior.java

```diff
@@ -170,6 +171,9 @@ public UpdateCallbackUrlResponse updateCallbackUrl(UpdateCallbackUrlRequest requ
             throw localizationProvider.buildExceptionForCode(ServiceError.INVALID_URL_FORMAT);
         }
 
+        // Remove this entity from REST client cache
+        restClientCache.remove(entity.getId());
```

Comment:
👀 getRestClient is synchronized, this is not. Isn't it a problem?

Reply:
Indeed, I already noticed and fixed in the next commit.